### ISSUE TITLE
When approaching a destination, display the number and street name onscreen

### ISSUE THREAD

**Issue by bb0x88** [organicmaps/organicmaps#9201]
As I approach a destination i often want to see the street name and number, it would be ideal to have the number and street name display when approaching a destination, for a reminder, maybe within 100m of the destination

**Comment by biodranik** [comment]
Where do you expect to see it? How other apps are showing it?

**Comment by bb0x88** [comment]
I would expect it to show up at the top of the screen, need to investigate other apps, but thats what I would imagine.

**Comment by muralito** [comment]
> As I approach a destination i often want to see the street name and number, it would be ideal to have the number and street name display when approaching a destination, for a reminder, maybe within 100m of the destination

You mean to display the same text you used when searching for destination? or maybe the address of the selected POI you choose to navigate to?

**Comment by bb0x88** [comment]
yes the address of the POI, because sometimes when I approach I forget the house number.  I then have to stop the car and look up what number house I was going to
